README.md

```diff
@@ -3,122 +3,205 @@
 
 ![videoroom_demo](https://membrane.stream/data/membrane_tutorials/videoroom/assets/records/expected_result.webp)
 
-## Environment variables
-Available runtime environment variables:
-```
-VIRTUAL_HOST={host passed to the endpoint config, defaults to "localhost" on non-production environments}
 
-USE_TLS={"true" or "false", if set to "true" then https will be used and certificate paths will be required}
-KEY_FILE_PATH={path to certificate key file, used when "USE_TLS" is set to true}
-CERT_FILE_PATH={path to certificate file, used when "USE_TLS" is set to true}
 
-INTEGRATED_TURN_IP={TURN server IP address}
-INTEGRATED_TURN_PORT_RANGE={port range, where UDP TURN will try to open ports. By default set to 50000-59999}
-INTEGRATED_TCP_TURN_PORT={port number of TCP TURN}
-INTEGRATED_TLS_TURN_PORT={port number of TLS TURN, used when "INTEGRATED_TURN_PKEY" and "INTEGRATED_TURN_CERT" are provided}
-INTEGRATED_TURN_PKEY={SSL certificate for TLS TURN}
-INTEGRATED_TURN_PKEY={SSL private key for TLS TURN}
-```
+## Running locally
 
-## Run manually
+### Installation of the dependencies
+To run phoenix application manually, you will need to have [`node.js`](https://nodejs.org/en/) installed. You can read about the installation [here](https://nodejs.org/en/download/).
+The demo has been tested with `node.js` version `v14.15.0`. 
 
-### Dependencies
+Furthermore, you will need some other dependencies.
+Depending on an operation system, you might also need to set up some
+environment variables. Below you can find a list of commands to
+be executed on a given OS.
 
-In order to run phoenix application manually you will need to have `node` installed.
-Demo has been tested with `node` version `v14.15.0`. You will also need some system dependencies.
+#### Mac OS X with Intel processor
 
-#### Mac OS X
+```
+brew install srtp clang-format ffmpeg
+export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
+export CFLAGS="-I/usr/local/opt/openssl@1.1/include/"
+export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include/"
+export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"
+```
+
+#### Mac OS X with M1 processor
 
 ```
 brew install srtp clang-format ffmpeg
+export C_INCLUDE_PATH=/opt/homebrew/Cellar/libnice/0.1.18/include:/opt/homebrew/Cellar/opus/1.3.1/include:/opt/homebrew/Cellar/openssl@1.1/1.1.1l_1/include
+export LIBRARY_PATH=/opt/homebrew/Cellar/opus/1.3.1/lib
+export PKG_CONFIG_PATH=/opt/homebrew/Cellar/openssl@1.1/1.1.1l_1/lib/pkgconfig/
 ```
 
+
 #### Ubuntu
 
 ```
 sudo apt-get install libsrtp2-dev libavcodec-dev libavformat-dev libavutil-dev
 ```
 
-### To run
-First install all dependencies:
+### Launching the demo
+First, install all project dependencies:
 ```
 mix deps.get
 npm ci --prefix=assets
 ```
 
-In order to run, type:
+To run, type:
```

Comment:
Why?

Reply:
Well, personally I'm keen on that expression, but that is what the grammarly says about it:
<img width="440" alt="image" src="https://user-images.githubusercontent.com/10851084/190435885-5fdcca2e-5efb-4cbb-b6db-fcac8ac78e65.png">
:(